### PR TITLE
Logout page

### PR DESCRIPTION
**Pull request by GuptaEesh**
Added a logout page.
A link to log in.

logout

### CODE REVIEW

File: cart/cart.html [hunk 1]
```diff
@@ -22,10 +22,10 @@
...
             </div>
-            <section>
-                <input type="text" class="input-text md" placeholder="Search...">
-                <a href="/login/login.html"><button class="primary btn md login without-shadow ">Login</button></a>
-            </section>
+            <input type="text" class="input-text md" placeholder="Search...">
+            <a href="/logout/logout.html"><button class="material-icons primary btn size-12 login without-shadow ">
+                    logout
+            </button></a>
             <div class="nav-section-right ">
```
Comment (Hamdrive) [inline cart/cart.html]: Formatting could be done better

**Review by Hamdrive — COMMENTED** [review]
Simple and clean code written. Will suggest to look into the formatting of your code to ensure it is easily readable.

File: cart/cart.html (hunk 1 above)
Comment (GuptaEesh) [reply cart/cart.html]: Will do that. Thanks for the suggestion.

**Review by Radheshyam1234 — COMMENTED** [review]
Your code looks clean with good use of variables.

**Comment by GuptaEesh** [comment]
Done with 2 reviews. Formatted the code properly following the suggestion of 1 reviewer. Now merging the branch.

**Pull request merged**